src/isomorphic/React.js

```diff
@@ -32,6 +33,21 @@ if (__DEV__) {
   cloneElement = ReactElementValidator.cloneElement;
 }
 
+var __spread = Object.assign;
+
+if (__DEV__) {
+  var warned = false;
+  __spread = function() {
+    warning(
+      warned,
+      'React.__spread is deprecated and should not be used. Use ' +
+      'Object.assign directly or another helper function with similar semantics'
+    );
+    warned = true;
+    return Object.assign.apply(Object, arguments);
```

Comment:
Going to make this `Object.assign.apply(null, arguments)` (shouldn't matter but makes more sense when compiling to use object-assign. `_assign.apply(null, arguments)